### PR TITLE
update link to table component

### PR DESCRIPTION
**Pull request by Djangofs**
**Checklist**:

* [ ] Documentation N/A
* [ ] Tests N/A
* [ ] Ready to be merged Yes! Simply a link change on a README

### CODE REVIEW

**Comment by penx** [comment]
Deploy preview for *govuk-react* processing.

Building with commit b074f5ed1aa084c24585ede4d007c2a04d7af1c0

https://app.netlify.com/sites/govuk-react/deploys/5c5c39deac2fb5000888dc78

File: components/table/README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ const example2Body = (
 ```
 
 ### References:
-- https://govuk-elements.herokuapp.com/alpha-beta-banners/
+- https://govuk-elements.herokuapp.com/data/
```
Comment (penx) [inline components/table/README.md]: should eventually be https://design-system.service.gov.uk/components/table/ but good to have it corrected for now 👍

**Comment by stevesims** [comment]
This change also needs to go into the corresponding index.js file as otherwise the readme file will get overwritten next time docs get regenerated

**Comment by Djangofs** [comment]
Have added the fix to index.js as well @stevesims

**Comment by stevesims** [comment]
Thanks @Djangofs - now merged 👍

**Pull request merged**